CONTRIBUTING.md

```diff
@@ -69,16 +69,16 @@ second to sign our CLA, which can be found
 
 Installing Go
 -------------
-InfluxDB requires Go 1.9.2.
+InfluxDB requires Go 1.10.3.
 
 At InfluxDB we find gvm, a Go version manager, useful for installing Go. For instructions
 on how to install it see [the gvm page on github](https://github.com/moovweb/gvm).
 
 After installing gvm you can install and set the default go version by
 running the following:
 
-    gvm install go1.9.2
-    gvm use go1.9.2 --default
+    gvm install go1.10.3
```

Comment:
I'm pretty sure nobody on our team uses gvm anymore. But we can deal with that doc change later.